configs/control4d-static.yaml

```diff
@@ -0,0 +1,120 @@
+name: "control4d-static"
+tag: "${rmspace:${system.prompt_processor.prompt},_}"
```

Comment:
consider adding `basename(data.dataroot)` to the tag so that it reflects which scene is working on